infra/gcp/ensure-staging-storage.sh

```diff
@@ -173,6 +173,11 @@ for REPO; do
         color 6 "Empowering ${WRITERS} to GCR"
         empower_group_to_write_gcr "${WRITERS}" "${PROJECT}"
 
+        # Enable prow-build Service Account to write to GCR
+        empower_svcacct_to_write_gcr \
+            $(svc_acct_email "k8s-infra-prow-build" "prow-build") \
+            "${PROJECT}"
+
```

Comment:
Do we want K8s Infra prow have access to write to _any_ staging GCR?

We might want a special case, explicitly for `gcr.io/k8s-staging-ci-images`, which can be added under this block:

https://github.com/kubernetes/k8s.io/blob/07a3a07626d8733550866abe799cf58241045a18/infra/gcp/ensure-staging-storage.sh#L230-L254

Reply:
Fixed.